test/system/280-update.bats

```diff
@@ -309,4 +309,22 @@ function nrand() {
 
     run_podman rm -t 0 -f $ctrname
 }
+
+# bats test_tags=ci:parallel
+@test "podman update - check resources over write Resources configuration" {
+    local ctrname="c-h-$(safename)"
+    run_podman run -d --name $ctrname \
+                --pids-limit 1024     \
+                $IMAGE /home/podman/pause
+
+    run_podman update $ctrname --memory 100M
+
+    run_podman inspect $ctrname --format {{.HostConfig.Memory}}
+    assert "$output" == "104857600" ".HostConfig.Memory"
+
+    run_podman inspect $ctrname --format {{.HostConfig.PidsLimit}}
+    assert "$output" == "1024" ".HostConfig.PidsLimit"
```

Comment:
you can combine these in one podman command 
`--format "{{.HostConfig.Memory}}\n{{.HostConfig.PidsLimit}}"` then match assert "${lines[0]}" ... and "${lines[1]}" respectively. This safes us one inspect command so it is a bit faster.